pom.xml

```diff
@@ -46,6 +46,9 @@
         <json.version>20201115</json.version>
         <fasterxml.jackson.core.version>[2.11.0,2.12.0)</fasterxml.jackson.core.version>
         <azure-storage.version>8.6.6</azure-storage.version>
+        <azure-identity.version>1.4.0</azure-identity.version>
+        <azure-core.version>1.21.0</azure-core.version>
+        <reactor-core.version>3.4.10</reactor-core.version>
```

Comment:
Why do we need this dependency?

Reply:
For the `block()`

Reply:
How does that work? you don't import it in the file, and java doesn't have extension methods